confgenerator/agentmetrics.go

```diff
@@ -41,6 +41,10 @@ func (r MetricsReceiverAgent) Pipeline() otel.Pipeline {
 			},
 		},
 		Processors: []otel.Component{
+			{
+				// perform custom transformations that aren't supported by the metricstransform processor
```

Comment:
I don't think this comment is true - I believe this is supported by `otel.AggregateLabels`

Reply:
Reimplemented using AggregateLabels, appears to work according to integration testing.